src/cms/utils/filter_links.py

```diff
@@ -0,0 +1,30 @@
+from django.db.models import Q
+
+from linkcheck.models import Link
+
+
+def filter_links(region_slug):
+    """[summary]
+
+    :param region_slug: [description]
+    :type region_slug: [type]
+    :return: [description]
+    :rtype: [type]
+    """
```

Comment:
Could you update these docstrings please?

Reply:
Yes, this was a mistake, I've updated comments to something more meaningful